tools/check_videos.py

```diff
@@ -132,28 +135,28 @@ def _do_check_videos(lock, dataset, output_file, idx):
     if os.path.exists(args.output_file):
         # remove exsiting output file
         os.remove(args.output_file)
-    pool = Pool(args.num_processes)
+    pool = Pool(cpu_count() - 1 or 1)
```

Comment:
Maybe we can keep `--num-processes` and set default value of `--num-processes` to `cpu_count() - 1`. Sometimes we cannot use all cpu cores.